tools/mermaid-js/Dockerfile

```diff
@@ -0,0 +1,15 @@
+# based on https://github.com/mermaid-js/mermaid.cli/blob/33a8334293308af13d307d5c3754841636e46da8/Dockerfile
```

Comment:
While the official repo does have a Dockerfile, there is, currently, no docker image on dockerhub, and also the Dockerfile doesn't work.